### DISCUSSION TITLE
Block for running an entire collection?

### DISCUSSION THREAD

**Discussion by RavenHursT** [postmanlabs/postman-flows#272] [Ideas]
So I can create blocks for running requests I've defined in my collections.  But is there some way we can get a block to just say "run all the requests in this collection"?

Basically looking for the same functionality that's currently in the "Run" button in collections already:
<img width="235" alt="image" src="https://user-images.githubusercontent.com/496983/157140989-bc6b9b70-3d37-4f70-bb8b-46c9a4a621ec.png">

**Comment by pablocoletto** [comment]
Or to have like a flow composition with steps from a collection already so it just makes everything in one block leaving current flow clear and clean. Also, helps to not repeat a lot of steps on every single flow
